pbsync/pbsync.py

```diff
@@ -35,27 +34,26 @@ def sync_handler(sync_val: str, repository_val=None, requested_bundle_name=None)
         remote_state, remote_url = pbgit.check_remote_connection()
         if not remote_state:
             pbtools.error_state(
-                "Remote connection was not successful. Please verify you have a valid git remote URL & internet connection. Current git remote URL: " + remote_url)
+                f"Remote connection was not successful. Please verify that you have a valid git remote URL & internet connection. Current git remote URL: {remote_url}")
         else:
             pblog.info("Remote connection is up")
 
         pblog.info("------------------")
 
-        pblog.info("Executing " + str(sync_val) + " sync command")
-        pblog.info("PBpy Module Version: " + pbpy_version.ver)
-        pblog.info("PBSync Executable Version: " + pbsync_version.ver)
+        pblog.info(f"Executing {sync_val} sync command")
+        pblog.info(f"PBpy Library Version: {pbpy_version.ver}")
+        pblog.info(f"PBSync Program Version: {pbsync_version.ver}")
 
         pblog.info("------------------")
 
         detected_git_version = pbgit.get_git_version()
         if detected_git_version == pbconfig.get('supported_git_version'):
```

Comment:
I'm getting this error while running sync all
```
10-May-20 18:36:22 [ERROR]  Git is not updated to the supported version in your system
10-May-20 18:36:22 [ERROR]  Supported Git Version: 2.26.2.vfs.1.2
10-May-20 18:36:22 [ERROR]  Current Git Version: 2.26.2.vfs.1.2
```

Reply:
I'm not sure what's the value on major part of this PR, but I feel like lots of stuff are broken.